File: tournaments/countSumOfTwoRepresentations/countSumOfTwoRepresentations.cpp
int countSumOfTwoRepresentations(int n, int l, int r) {
  int result = 0;

  for (int a = l; a <= r; a++) {
    for (int b = a; b <= r; b++) {
      if (a + b == n) {
        result++;
      }
    }
  }

  return result;
}

File: tournaments/countSumOfTwoRepresentations/countSumOfTwoRepresentations.js
function countSumOfTwoRepresentations(n, l, r) {
    let result = 0;

    for (let i = l; r >= i; ++i)
        for (let j = i; r >= j; ++j)
            if (n === i + j)
                ++result;

    return result;
}
